diff --git a/pkgs/by-name/ju/jujutsu/package.nix b/pkgs/by-name/ju/jujutsu/package.nix
index c096701c143e2..2ed77e0269426 100644
--- a/pkgs/by-name/ju/jujutsu/package.nix
+++ b/pkgs/by-name/ju/jujutsu/package.nix
@@ -35,2 +35,2 @@ rustPlatform.buildRustPackage {

-  cargoHash = "sha256-35DJdAUXc2gb/EXECScwinSzzp7uaxFbUxedjqRGfj8=";
+  cargoHash = "sha256-b+BJI92f+/GAwzvAdJSEn/9rnTQj3l0HQhNTPybOA/4=";
